content/applications/inventory_and_mrp/inventory/management/misc/batch_transfers.rst

```diff
@@ -1,124 +1,223 @@
 
-In the example below, a filter was applied to only see the transfers that are in the *Pick* step.
-After that, the different transfers that needed to be included in the batch transfer were selected.
+.. image:: batch_transfers/set-2-or-3-step-shipment.png
+   :align: center
+   :alt: Set up 2-step or 3-step outgoing shipments.
 
-.. image:: batch_transfers/batch_transfers_04.png
-    :align: center
-    :alt: View of the list of transfers to choose for a single batch transfer and how to add them
-          to the batch transfer
+Create batch transfers
+======================
 
-To see the products to pick for the different transfers, click on
-*Select*. If *Multi-locations* has been activated, the document also shows the locations they have
-been reserved from.
+Manually create batch transfers directly from the :menuselection:`Inventory app --> Operations -->
+Batch Transfers` page. Click the :guilabel:`New` button to begin creating a batch transfer.
 
-.. image:: batch_transfers/batch_transfers_05.png
-    :align: center
-    :alt: View of a batch transfer list. Products to pick with their source and target locations
+On the batch transfer form, fill the following fields out accordingly:
 
-Create a Batch Transfer from the Transfers List View
-====================================================
+- :guilabel:`Responsible`: employee assigned to the picking. Leave this field blank if any worker
+  can fulfill this picking.
+- :guilabel:`Operation Type`: select the operation type the picking is categorized under from the
+  drop-down menu.
+- The :guilabel:`Scheduled Date` at the top of the form specifies the date the
+  :guilabel:`Responsible` person should complete the transfer to the output location by.
 
-From the *Transfers List View*, select transfers that should be
-included in the Batch. Then, select *Add to batch* from the *Action*
-list.
+Next, in the :guilabel:`Transfers` list, click :guilabel:`Add a line` to open the :guilabel:`Add:
+Transfers` window. If the :guilabel:`Operation Type` field was filled, the list will filter transfer
+records matching the selected :guilabel:`Operation Type`. Click the :guilabel:`New` button to create
+a new transfer. Once the transfer records are selected, click :guilabel:`Confirm` to confirm the
+batch picking.
 
-.. image:: batch_transfers/batch_transfers_06.png
-    :align: center
-    :alt: View of the process to add transfers to a batch transfer from the transfers list view
+.. note::
+   For reviewers; Remove this before publishing (:
+   Create a SO and confirm it after setting the Outgoing shipment option to 2 or 3 steps to generate
+   a Pick entry for the example in runbot. This is because if the runbot was originally configured
+   in 1-step, there would be no entries in the newly created operation type, :guilabel:`Pick`!
 
-Next, determine if you want to add the transfers to an existing draft
-batch transfer or create a new one.
+.. example::
```

Comment:
```suggestion

```

Reply:
moved this below, but can start the entire example admonition on line 79, but proper formatting